data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+	description "Go to <destination> and learn more about the Hai's worlds."
+	source "Allhome"
+	destination "Stonebreak"
+	to offer
+		or
+			has "ZZ Event Hai Secret Leaks: RS: done"
+			has "ZZ Event Hai Secret Leaks: FW: done"
+	on offer
+		conversation
+			`As you are leaving the Merchant and the Hai, the Hai calls out as an afterthought, "Oh Captain, if you really want to see what the Hai are like, you should go visit Stonebreak."`
+			choice
+				`	"Thanks, I'll do that!"`
+					accept
+
+	#Goto Stop 2 Alta Hai (Alta Hai Ringworld)
+mission "Hai Tour [2] Alta Hai"
+	landing
+	name "Visit the ringworld on <planet>"
+	description "Go to <destination> and learn more about the Hai's worlds."
+	source "Stonebreak"
+	destination "Alta Hai"
+	to offer
+		has "ZZ Hai Tour [1] Stonebreak: done"
+	on offer
+		conversation
+			`Stonebreak is home to some of the largest cities in Hai space aside from their homeworld: massive planned communities built in concentric rings, where housing space and community buildings alternate with zones of factories and shipyards. This star system has an unusual number of metallic asteroids, which the Hai harvest for the raw materials for Stonebreak's industries.`
+			`	From above, this city has almost perfect radial symmetry, a sure sign that it was planned and built as a single unit rather than expanding at random over time. Every single building in the city is made from the same metal alloy, a dull and slightly irregular grey color. The corners of each building are slightly beveled instead of meeting at right angles.`
+			`	Even the spaceport is symmetrical: a ring of hangars, facing outwards, at evenly spaced intervals, with warehouses and living quarters inside the ring.`
+			`	While the layout is utilitarian, it is an incredibly comfortable place to live, and the Hai have given their individual community spaces quite a lot of character and life. It is, overall, an excellent example of an idealised Hai world, built in the presence of abundant resources.`
+			`	As you are wandering around and getting a feel for the place you overhear someone talking about an apparently complete ringworld in Alta Hai. It definitely sounds like something you should check out.`
+				accept
+
+	#Goto Stop 3 Wah Yoot (Darkcloak) - must happen after Quarg first contact
+mission "ZZ Hai Tour [3] Meet Unfettered"
+	landing
+	name "Meet the Rebels on <planet>"
+	description "Go to <destination> and learn more about the Hai's conflict."
+	source "Alta Hai"
+	destination "Darkcloak"
+	to offer
+		has "Hai Tour [2] Alta Hai: done"
+	on offer
+		conversation
+			`This is a complete Quarg ringworld, the product of millennia of construction effort which has consumed every other stellar object in this system in order to provide the raw materials. The inner edge of the ring is filled with solar panels, hydroponic gardens, living quarters, and sunlit common spaces. The outer edge contains hangars and docks for ships, and storage facilities for cargo.`
```

Comment:
I asked about that last sentence in the discord [here](https://discordapp.com/channels/251118043411775489/266345072554016768/746142672959569951). Below is Marksman's recommendation for the line.
```suggestion
			`This is a complete Quarg ringworld, the product of millennia of construction effort which has consumed every other stellar object in this system in order to provide the raw materials. The inner edge of the ring is filled with solar panels, hydroponic gardens, living quarters, and sunlit common spaces. The outer edge contains hangars and docks for ships while housing storage facilities for cargo.`
```